logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/team/GetTeamUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.team
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.team.TeamRepository
+import com.wire.kalium.logic.functional.Either
+
+class GetTeamUseCase(private val teamRepository: TeamRepository) {
```

Comment:
```suggestion
class GetSelfTeamUseCase(private val teamRepository: TeamRepository) {
```
This use case can be part of `GetSelfUserUseCase` or `SyncSelfUserUseCase`

Reply:
During our conversation I have removed `GetTeamUseCase` and adjusted the `fetchTeamInfo()` function to do exactly one thing and it is now being used in `UserRepository` so it can be correctly fetched with the user info (as we need the teamId from the user)